src/pages/Thanks/ThanksPage.vue

```diff
@@ -334,6 +334,13 @@ export default {
 			}) || {};
 
 			this.shareCardLanguageVersion = shareCardLanguage.version;
+			if (this.shareCardLanguageVersion) {
+				this.$kvTrackEvent(
+					'Thanks',
+					'EXP-MARS-143-Jul2022',
```

Comment:
These are the same tracking details used on the borrower profile. Should we add something to this so that it's easier to differentiate these events? Like, `EXP-MARS-143-Jul2022-inviter`

Reply:
Yeah, will do that!

Reply:
done!